stixcore/tmtc/parser.py

```diff
@@ -0,0 +1,173 @@
+from collections import defaultdict
+
+from bitstring import ConstBitStream
+
+from stixcore.util.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+class PacketData:
+    """Generic class for organizing all parameters of the TM packet data."""
+
+
+def parameter_dict_2_PacketData(d):
```

Comment:
Maybe make this a class method on the `PacketData` class?

Reply:
good point